web/themes/joinup/templates/content/node--document--listing-item.html.twig

```diff
@@ -107,7 +107,7 @@
     {% endif %}
     <div class="listing__field listing__stats mdl-card__actions listing__action-link__wrapper action-link__wrapper">
       <a href="{{ url }}" class="listing__action-link action-link">{{ 'Details'|t }}</a>
-      <a href="{{ content.field_file|render|striptags|trim }}" class="listing__action-link action-link">{{ 'Download'|t }}</a>
+      <a href="{{ content.field_file.0|render|striptags|trim }}" class="listing__action-link action-link">{{ 'Download'|t }}</a>
```

Comment:
Hi @tom-waat the fix is handling the case for 1 file.
One suggestion: use a small preprocess as [this one](https://github.com/ec-europa/joinup-dev/blob/e08f22e5dae4f7e5b39d480686d33e1d11e3863b/web/themes/joinup/joinup_theme.theme#L919) to pass a the var in a bit better way. The current fix is ok for the moment, but is a bit cheese and can cause confusion later if more files are to be displayed.